Elixir/tools/chocolateyInstall.ps1

```diff
@@ -15,10 +15,11 @@ if (!(Test-Path($params.UnzipLocation)))
 
 Install-ChocolateyZipPackage @params
 
-$elixirPath = "$env:ChocolateyPackageFolder/bin"
```

Comment:
I know that no one believes me but "/" is a valid path separator on Windows too.  That's not a mistake which needs to be corrected.